<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Gorod_b" name="com.example.tests.TestContactCreation" tests="1" failures="1" timestamp="28 Sep 2014 09:21:11 GMT" time="0.052" errors="0">
  <testcase name="shouldCreateContactWithValidData" time="0.052" classname="com.example.tests.TestContactCreation">
    <failure type="com.jacob.com.ComFailException" message="Can&#039;t get object clsid from progid">
      <![CDATA[com.jacob.com.ComFailException: Can't get object clsid from progid
	at com.jacob.com.Dispatch.createInstanceNative(Native Method)
	at com.jacob.com.Dispatch.<init>(Dispatch.java:99)
	at com.jacob.activeX.ActiveXComponent.<init>(ActiveXComponent.java:58)
	at autoitx4java.AutoItX.<init>(AutoItX.java:181)
	at com.example.framework.AutoItHelper.<init>(AutoItHelper.java:20)
	at com.example.framework.ApplicationManager.getAutoItHelper(ApplicationManager.java:63)
	at com.example.framework.ContactHelper.initContactCreation(ContactHelper.java:16)
	at com.example.framework.ContactHelper.createContact(ContactHelper.java:10)
	at com.example.tests.TestContactCreation.shouldCreateContactWithValidData(TestContactCreation.java:13)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- shouldCreateContactWithValidData -->
</testsuite> <!-- com.example.tests.TestContactCreation -->
